fisco-bcos/para/CMakeLists.txt

```diff
@@ -0,0 +1,31 @@
+#------------------------------------------------------------------------------
+# Link libraries into main.cpp to generate executable binrary fisco-bcos
+# ------------------------------------------------------------------------------
+# This file is part of FISCO-BCOS.
+#
+# FISCO-BCOS is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# FISCO-BCOS is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with FISCO-BCOS.  If not, see <http://www.gnu.org/licenses/>
+#
+# (c) 2016-2018 fisco-dev contributors.
+#------------------------------------------------------------------------------
+
+
+file(GLOB SRC_LIST "*.cpp")
+file(GLOB HEADERS "*.h")
+    set(CMAKE_CXX_FLAGS_DEBUG          "-Og -g -pthread -DETH_DEBUG -fopenmp")
```

Comment:
I think this option should in compilerSetting.cmake

Reply:
deleted